src/base/bittorrent/session.cpp

```diff
@@ -4129,6 +4131,10 @@ quint64 Session::getAlltimeUL() const
 
 void Session::refresh()
 {
+    if (lt::clock_type::now() - m_statsLastTimestamp < MINIMUM_STATS_UPDATE_INTERVAL) {
```

Comment:
pls add parentheses: `(lt::clock_type::now() - m_statsLastTimestamp)`

Reply:
This code looks confusing.
Why not just limit corresponding setting to be non less than minimum interval?

Reply:
> Why not just limit corresponding setting to be non less than minimum interval?

corresponding setting (Refresh interval) is already limited but this branch is still triggered
"Attempt to update...." was set to be triggered inside this branch
![image](https://user-images.githubusercontent.com/34717789/88451266-e3b36580-ce72-11ea-95be-dde3a9678e1c.png)